src/components/CreateCourse/CreateCourse.jsx

```diff
@@ -75,11 +109,11 @@ const CreateCourse = ({
 		event.preventDefault();
 		if (!authorInput.current.value) return;
 		const newAuthor = {
-			id: uuidv4(),
 			name: authorInput.current.value,
 		};
-		addNewAuthors((authorsList) => [...authorsList, newAuthor]);
-		setAuthorList((authorList) => [...authorList, newAuthor]);
+
+		fetchWithToken('authors/add', newAuthor, token);
+		fetchAuthors();
```

Comment:
I'd not recommend to add author and refetch authors list in the same time: it may end up with the situation, when list of authors doesn't have newly created author